lib/app.rb

```diff
@@ -161,4 +168,46 @@ def add_book
     puts 'Book created successfully!'
     @books << book
   end
+
+  def list_musicalbum
+    if @musicalbums.length.zero?
+      puts 'No Music Album added yet !'
+    else
+
+      @musicalbums.each_with_index do |album, index|
+        puts "#{index + 1} Music Album :"
+        puts album
```

Comment:
Optional: Kindly print the results in a single line (just like you did with genre) so the user is not confused.